annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -248,7 +254,7 @@ protected void validateSemanticTagVcardAddressBody(Body body)
      * @throws ParamValidationException
      */
     protected void validateTextualBody(Body body, boolean isLanguageMandatory) throws ParamValidationException {
-
+    if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.